### DISCUSSION TITLE
RichEditor goes blank when switching locales with Spatie Translatable plugin

### DISCUSSION THREAD

**Discussion by christopheraseidl** [filamentphp/filament#11873] [Help]
### Package

Panel builder

### Package Version

v3.2.50

### How can we help you?

Although TextInput and MarkdownEditor components translate fine when switching locales, something is causing the value of the RichEditor to go blank after switching. When switching locales, I can briefly see the English/Spanish versions appear before going blank, and then it is blank for good. I've tried looking into the source code and also starting a new project with the same result and can't for the life of me figure out what is happening. (I also uploaded videos of what's happening but Github says the file is corrupt?).

**Comment by hieulvt** [comment]
I encounter the same issue with RichEditor. Have you found any solution @christopheraseidl ?

**Reply by christopheraseidl** [reply]
Unfortunately, I have not. You might [have a look at this post](https://github.com/filamentphp/filament/issues/11175#issuecomment-2103480974) on the subject of translation troubles, however.
